### PR TITLE
TST: Test CoW with DataFrame.items()

### PR DESCRIPTION
**Pull request by lithomas1**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/tests/copy_view/test_methods.py [hunk 1]
```diff
@@ -721,3 +721,22 @@ def test_squeeze(using_copy_on_write):
         # Without CoW the original will be modified
         assert np.shares_memory(series.values, get_array(df, "a"))
         assert df.loc[0, "a"] == 0
+
+
+def test_items(using_copy_on_write):
+    df = DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]})
+    df_orig = df.copy()
+    for name, vals in df.items():
+        ser = Series(vals)
```
Comment (mroeschke) [inline pandas/tests/copy_view/test_methods.py:730]: Does `vals` need to be rewrapped in `Series`? vals is already a `Series`

**Comment by jorisvandenbossche** [comment]
@lithomas1 can you update this for Matt's small comment?

File: pandas/tests/copy_view/test_methods.py (hunk 1 above)
Comment (lithomas1) [reply pandas/tests/copy_view/test_methods.py]: Thanks updated.

**Review by jorisvandenbossche — APPROVED** [review]
Thanks!

**Pull request merged**